eng/pipelines/docindex.yml

```diff
@@ -46,10 +46,11 @@ jobs:
           pwsh: true
           filePath: eng/common/scripts/Service-Level-Readme-Automation.ps1
           arguments: >-
-            -DocRepoLocation $(DailyDocRepoLocation)
+            -DocRepoLocation $(DocRepoLocation)
```

Comment:
We should also remove the entry at https://github.com/Azure/azure-sdk-for-java/pull/30175/files#diff-f1e244733dfe16323f099542a3b4a1ba8bb68f45cf329d8dac9d1613c5956c40R12 unless it's still needed.

Reply:
Nevermind I see it used in https://github.com/Azure/azure-sdk-for-java/pull/30175/files#diff-f1e244733dfe16323f099542a3b4a1ba8bb68f45cf329d8dac9d1613c5956c40R112